props/saber_fett263_buttons.h

```diff
@@ -257,7 +335,104 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
         previous_preset();
         return true;
-	
+
+       // Gesture Controls
+		
+       // Battle Mode
+		    
+#ifdef FETT263_BATTLE_MODE
+  #define FETT263_SWING_ON
+  #define FETT263_TWIST_OFF
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_AUX):
+         if (!battle_mode_) {
+           battle_mode_ = true;
+           // Force sound plays when entering Battle Mode
+           hybrid_font.SB_Force();
+         } else {
+           battle_mode_ = false;
+	   // Exit Color Change sound plays when exiting Battle Mode
+           hybrid_font.SB_Change(EXIT_COLOR_CHANGE);
+         }
+         return true;
+
+       // Auto Lockup Mode
+       case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_ON):
+         if (!battle_mode_) return false;
+         clash_impact_millis_ = millis();
+         swing_blast_ = false;
+         if (!swinging_) {
+           SaberBase::SetLockup(SaberBase::LOCKUP_NORMAL);
+           swing_blast_ = false;
+           auto_lockup_on_ = true;
+           SaberBase::DoBeginLockup();
+           return true;
+	 }
+
```

Comment:
shouldn't there be a return here?

Reply:
added

Reply:
Now you can return the return from two lines up though, as they are the same.

Reply:
I see, done.